backend/market/views.py

```diff
@@ -1,37 +1,48 @@
 from django.contrib.auth import get_user_model
-from django.db.models import prefetch_related_objects
 from django.utils import timezone
 from rest_framework import exceptions, generics, mixins, status, viewsets
 from rest_framework.generics import get_object_or_404
 from rest_framework.parsers import FormParser, MultiPartParser
 from rest_framework.permissions import IsAuthenticated
 from rest_framework.response import Response
 
-from market.models import Amenity, Offer, Sublet, SubletImage
+from market.models import Category, Item, ItemImage, Offer, Sublet, Tag
 from market.permissions import (
     IsSuperUser,
+    ItemImageOwnerPermission,
+    ItemOwnerPermission,
     OfferOwnerPermission,
-    SubletImageOwnerPermission,
     SubletOwnerPermission,
 )
 from market.serializers import (
-    AmenitySerializer,
+    CategorySerializer,
+    ItemImageSerializer,
+    ItemImageURLSerializer,
+    ItemSerializer,
+    ItemSerializerSimple,
     OfferSerializer,
-    SubletImageSerializer,
-    SubletImageURLSerializer,
     SubletSerializer,
-    SubletSerializerRead,
-    SubletSerializerSimple,
+    TagSerializer,
 )
 from pennmobile.analytics import Metric, record_analytics
 
 
 User = get_user_model()
 
 
-class Amenities(generics.ListAPIView):
-    serializer_class = AmenitySerializer
-    queryset = Amenity.objects.all()
+class Tags(generics.ListAPIView):
```

Comment:
hm for these (Tags and Categories) lets just make this a normal APIView with 1 get request chuck out the queryset and serializer_class, and just do like `return Tags.objects.all().values_list(flat=True)`